web/styles/dark_theme.css

```diff
@@ -784,8 +784,8 @@
         }
     }
 
-    #user_presences li:hover,
-    #user_presences li.highlighted_user {
+    #narrow-user-presences li:hover,
```

Comment:
This looks like a good opportunity to set CSS variables on the colors here and below, and get further out of the business of duplicate selectors in this dark theme file.

Reply:
I don't want to do it in the same commit, since that was just renaming selectors, but I'm happy to put a commit on the end of this PR with a variable! Great idea :)

Reply:
Oh actually there's a bug because I never made an "other users" version of this in the dark theme file! Exactly why variables are better :)